projects/packages/forms/src/blocks/field-textarea/edit.js

```diff
@@ -9,8 +9,8 @@ import useJetpackFieldStyles from '../shared/hooks/use-jetpack-field-styles';
 import { ALLOWED_INNER_BLOCKS } from '../shared/util/constants';
 
 export default function TextareaFieldEdit( props ) {
-	const { attributes, clientId, id, isSelected, label, requiredText, setAttributes } = props;
```

Comment:
There's some hallucinated props here. I think this is copy/pasta or AI code.

Should they be from `attributes` instead?

Reply:
Yeah I think these are copy/pasted from the old state of the text area edit component, because previously the `FieldDefaults` included these attributes (`id`, `label`, `requiredText`). In `trunk`, they're included in `FieldDefaults`: https://github.com/Automattic/jetpack/blob/90f41fdd092ec8b6ae94b622d10f8ec42fdf1f21/projects/packages/forms/src/blocks/contact-form/child-blocks.js#L24